share/spack/gitlab/cloud_pipelines/stacks/build_systems/spack.yaml

```diff
@@ -1,5 +1,11 @@
 spack:
   view: false
+
+  include:
+  - $spack/share/spack/gitlab/cloud_pipelines/configs/linux/x86_64_v3
+  - $spack/share/spack/gitlab/cloud_pipelines/configs/linux
+  - $spack/share/spack/gitlab/cloud_pipelines/configs
```

Comment:
This is unexpected to me, is it related to windows ci or an artifact of the rebase or testing or something?

Reply:
@kwryankrattiger this is from commit https://github.com/spack/spack/pull/43967/commits/9f3f40c95ad89f440697f72c96efd15216b21619, I assume it's a refactor because the Windows side of configuration made the previous scope behavior no longer sufficiently generic?

Reply:
This will come in another PR someday. I think I did this because the windows ci PR was originally branched off of another refactor branch I was playing with.

Reply:
So revert all the include changes?

Reply:
yeah, you should be able to drop all of those. This is redundant with the `-C` stuff in the `.gitlab-ci.yml`.

I will make a  point of making another PR to do this later, the current design is kind of unfortunate.

Reply:
Ugh, see other comment...

Reply:
By `-C` is assume you mean the `--config-scope` changes, in which case, addressed.